src/contracts/URMCore.sol

```diff
@@ -485,4 +476,48 @@ contract UpgradeRegressionManager is IUpgradeRegressionManager {
     // Rollback is proposed but not queued for execution.
     return IGovernor.ProposalState.Pending;
   }
+
+  /*///////////////////////////////////////////////////////////////
+                      Target Interaction Hooks
```

Comment:
Not sure I like this title. To me a hook means something else. I would just label it "abstract methods" or something.